masterlist.yaml

```diff
@@ -17632,12 +17910,15 @@ plugins:
       - 'URN+CoT Patch.esp'
       - 'Atlas Map Markers.esp'
       - 'The Horkerpocalypse.esp'
-    tag:
-      - C.Location
```

Comment:
Ah, I missed that. I thought it was removed entirely like for https://github.com/loot/skyrim/pull/136#discussion_r122011799